guides/common/modules/proc_exporting-templates.adoc

```diff
@@ -15,24 +15,30 @@ The {ProjectWebUI} displays the status of the export.
 The status is not persistent; if you leave the status page, you cannot return to it.
 
 .CLI procedure
-. Clone a local copy of your Git repository:
+. Clone a local copy of your Git repository to `/var/lib/foreman`:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
-$ git clone https://github.com/theforeman/community-templates __/custom/templates__
+$ git clone https://github.com/theforeman/community-templates /var/lib/foreman
 ----
-. Change the owner of your local directory to the `foreman` user, and change the SELinux context with the following commands:
++
+[NOTE]
+====
+You can clone your repository to a custom directory outside `/var/lib/foreman`, but you have to set the `foreman_lib_t` SELinux label to that directory.
```

Comment:
"… but you have to ensure the Foreman service can read its contents (check file permissions and SELinux label being `foreman_lib_t`)"

or something along the lines -- SELinux is not the only one who will prevent things from working,

Reply:
Thank you, applied.